android/src/main/kotlin/io/vinicius/appodeal_flutter/AppodealFlutterPlugin.kt

```diff
@@ -2,17 +2,9 @@ package io.vinicius.appodeal_flutter
 
 import android.app.Activity
 import androidx.annotation.NonNull
-import com.appodeal.ads.Appodeal
-import com.appodeal.ads.BannerCallbacks
-import com.appodeal.ads.InterstitialCallbacks
-import com.appodeal.ads.NonSkippableVideoCallbacks
-import com.appodeal.ads.RewardedVideoCallbacks
-import com.explorestack.consent.Consent
+import com.appodeal.ads.*
```

Comment:
Can you please use individual imports instead of wildcard imports?

I know this is one of those debates, like tabs vs spaces, and in my opinion there's no specific advantage or disadvantage to use on method over the other, but since I adopted the code style in the project to import each class individually, then I would like to stick with this code style in all files in the project 🙂